docs/components/Attribute/basic_usage.rst

```diff
@@ -104,15 +104,15 @@ of the :ref:`component_attribute_model_attribute-subject-interface`.
        /**
         * {@inheritdoc}
         */
-       public function hasAttributeByCode($attributeCode)
+       public function hasAttributeByCodeAndLocale($attributeCode, $localeCode)
```

Comment:
We could perhaps improve the DX here a bit and make the 2nd parameter optional. If not provided,  the entity would use the ``currentLocale`` stored on it at the moment of call. WDYT?